packages/ui-components/src/components/step-bar/test/step-bar.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import { newSpecPage, SpecPage } from '@stencil/core/testing';
+import { h } from '@stencil/core';
+import { STEPS_ERROR_MOCK, STEPS_SUCCESS_MOCK } from './step-bar.mock';
+import { KvStepBar } from '../step-bar';
+
+describe('KvStepBar (unit tests)', () => {
+	let page: SpecPage;
+	let component: KvStepBar;
+
+	describe('when rendering with required props', () => {
+		beforeEach(async () => {
+			page = await newSpecPage({
+				components: [KvStepBar],
+				template: () => <kv-step-bar steps={STEPS_SUCCESS_MOCK} currentStep={1} progressPercentage={50}></kv-step-bar>
+			});
+			component = page.rootInstance;
+		});
+
+		it('should match the snapshot', () => {
+			expect(page.root).toMatchSnapshot();
+		});
+
+		it('should set the correct steps array', () => {
```

Comment:
Not sure if this makes sense since it's the exact prop you are passing. If you do this for this prop why not do it for all the props? What would be the purpose of doing this?

Reply:
I'll go ahead and remove that assertion given that it's doing absolutely nothing that confirms content being rendered on the screen.

Our current approach on testing with snapshots requires us to review the snapshots on a PR, as Jest recommends [here](https://jestjs.io/docs/snapshot-testing#1-treat-snapshots-as-code), which means that these "render" assertions must be made "manually" by us as a part of the review process.

I'll see if I can find some time to investigate what testing approaches suit our needs best for this lib like we'd discussed ;)